client/src/plus/collections/v2/collection.tsx

```diff
@@ -228,8 +240,22 @@ function ItemComponent({
             </div>
           </div>
         </MDNModal>
+      </header>
+      <div className="breadcrumbs">
+        {item.parents
+          .slice(0, -1)
+          .map(
+            (parent, index, parents) =>
+              parent.title !== parents[index + 1]?.title &&
+              (index ? " > " : "") + parent.title
+          )}
```

Comment:
I feel like this map callback is hard to understand/read, so maybe it would make sense to extract this to a variable `breadcrumbs`, and just `breadcrumbs.join(" > ")` here.